themes/default/content/learn/building-with-pulumi/secrets/index.md

```diff
@@ -170,7 +214,7 @@ config:
 We can access the secrets similarly to other configuration data, however we must
 specify that it is a secret. Add this code to {{< langfile >}} inside of `my-first-app`:
```

Comment:
Not this PR, but this should probably use "replace the code" here instead, since adding doesn't quite work.

Reply:
Changed in c54cc22